### PR TITLE
Fix for nmea hdt message not being sent [MAP-691] Master

### PR DESCRIPTION
**Pull request by denniszollo**

### CODE REVIEW

**Comment by benjaminaltieri** [comment]
@denniszollo can you add the JIRA ticket ID to PR title? thanks!

**Comment by nicolekelley** [comment]
@denniszollo - are we merging this?

File: package/sbp_nmea_bridge/src/sbp_nmea_bridge.c [hunk 1]
```diff
@@ -151,6 +152,24 @@ static void gps_time_callback(u16 sender_id, u8 len, u8 msg[], void *context)
...
+static void baseline_heading_callback(u16 sender_id, u8 len, u8 msg[], void *context)
+{
+  (void)context;
+  (void)sender_id;
+  (void)len;
+  msg_baseline_heading_t *baseline_hdg = (msg_baseline_heading_t *)msg;
+  sbp2nmea_baseline_heading(baseline_hdg, &state);
+}
+
+static void baseline_heading_callback(u16 sender_id, u8 len, u8 msg[], void *context)
```
Comment (nsirola) [inline package/sbp_nmea_bridge/src/sbp_nmea_bridge.c]: Hmm duplicate function, does not compile.

Comment (nsirola) [reply package/sbp_nmea_bridge/src/sbp_nmea_bridge.c]: Removing the extra one in https://github.com/swift-nav/piksi_buildroot/pull/926

**Comment by silverjam** [comment]
@denniszollo please use squash when merging in the future

**Pull request merged**